src/pages/configurePatch/ConfigurePatchCore.tsx

```diff
@@ -267,10 +265,10 @@ const filterAliases = (
   return patchTriggerAliases.filter(({ alias }) => !invokedAliases.has(alias));
 };
 
-const StyledInput = styled(Input)`
+const StyledInput = styled(TextInput)`
   margin-bottom: ${size.s};
-  font-weight: 600;
-`;
-const StyledBody = styled(Body)`
-  margin-bottom: ${size.xxs};
+  input {
+    font-weight: 600;
+    font-size: 16px; // Remove when TextInput is updated, since there is a new props baseFontSize.
```

Comment:
Is this necessary? Otherwise it looks different than our other text inputs?

Reply:
I changed this to be an inline style so it's more obvious to whoever eventually updates the props for the upgraded `TextInput`. It seems like the new `TextInput` explicitly supports `16px` base font size according to its [props](https://www.mongodb.design/component/text-input/documentation/), so I think it's okay